### ISSUE TITLE
[Bug]: ARM-version does't run on MacOS 15.3.1 (24D70)

### ISSUE THREAD

**Issue by Atraides59** [FreeTubeApp/FreeTube#6920]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have [searched the issue tracker for open and closed issues](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sort%3Arelevance-desc) that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

The installation of the ARM-version fails with the following (translated) error message:

"FreeTube" is corrupted and cannot be opened. It is recommended that you move the item to the Trash.

Workaround is to use the x86-version with the known tricks to stop macOS from complaining. Same tricks for the arm64 version don't work and leads to the above error message.

### Expected Behavior

arm64 version should start like the x86 version does.

### Issue Labels

inconsistent behavior, accessibility issue

### FreeTube Version

v0.23.2-arm64

### Operating System Version

MacOS 15.3.1 (24D70) on Mac Mini M4 Pro

### Installation Method

.dmg

### Primary API used

Invidious API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by absidue** [comment]
https://docs.freetubeapp.io/faq/#macos-freetube-is-damaged-and-cant-be-opened-you-should-move-it-to-the-trash
